src/app/chat/features/TopicListContent/Header.tsx

```diff
@@ -78,7 +78,12 @@ const Header = memo(() => {
           </Dropdown>
         </>
       }
-      title={`${t('topic.title')} ${topicLength > 1 ? topicLength + 1 : ''}`}
+           title={
```

Comment:
I get ya, though I think it looks odd to have something like "Topic List 9", I think there it looks cleaner with a Tag and makes it clear that its a counter. I was confused at first when it got added.
![image](https://github.com/lobehub/lobe-chat/assets/75225859/fab91a11-7bdc-4c8a-9180-7b58eeec52d4)

Reply:
@canisminor1990 what's your idea?

Reply:
@LeEricCH We have discussed about this update, and we think to retain the preview design and accpect your locale change. So it will display as "Topics 9",and we think it's clean enough.

please just keep your locale update. Thanks! 🙏